src/main/webapp/scripts/analyze.js

```diff
@@ -0,0 +1,7 @@
+function analyzeVideo() {
+  const videoId = document.getElementById("video-url").value.split('v=')[1];
+  fetch('/sentiment', {
+    method: 'POST',
+    body: videoId
+  });
+}
```

Comment:
Keep JS changes to a different PR unless it is required due to breaking/changing a previous feature..